.travis.yml

```diff
@@ -1,8 +1,15 @@
 # The language in this case has no bearing - we are going to be making use of conda.
 language: generic
 
-os: osx
-osx_image: xcode6.4
+jobs:
+  include:
+    - if: (fork IN (true)) OR (type IN (pull_request))
+      os: osx
+      osx_image: xcode6.4
+
+    - if: (fork IN (false)) AND (NOT (type IN (pull_request)))
+      os: linux
+      sudo: false
```

Comment:
We should be able to have `sudo: false` regardless, right? Admittedly I don't think it matters much for macOS, but still.

Reply:
`sudo: false` is there to select which type of linux container. Not used for`OSX at all

Reply:
Right. Though we do have `sudo: false` a few lines below already. Maybe we should drop one of these lines.